src/components/Clock/Clock.module.scss

```diff
@@ -1,7 +1,33 @@
-.clock {
-    width: 483px;
-    height: 478px;
-
-    background-image: url(../../images/clock.jpg);
-    border-radius: 50%;
+.clock {
+    width: 480px;

```

Comment:
please, introduce scss variable to keep width/height, because they are always the same